fuzzer/modules/sql_injection/run_lab_6.py

```diff
@@ -0,0 +1,6 @@
+from strategies.union_strategy import lab_6_union_attack_test
+
+BASE_URL = 'http://127.0.0.1:4000'
+
+
+lab_6_union_attack_test(BASE_URL)
```

Comment:
This essentially duplicates lab6 to demonstrate how it is a difficult test to automate.